### ISSUE TITLE
Plugin doesn't work with @font-face

### ISSUE THREAD

**Issue by visionpixel** [cuth/postcss-pxtorem#14]
This great plugin doesn't work with the @font-face declaration.

My config is an empty array in prop_white_list.
selector_black_list doesn't filter the font-face.

Here is the error:
```
TypeError: Cannot call method 'match' of undefined
    at node_modules/postcss-pxtorem/index.js:72:25
    at Array.some (native)
    at blacklistedSelector (node_modules/postcss-pxtorem/index.js:71:22)
```

Greetings

**Comment by cuth** [comment]
The fix is in version 2.3.2 and published to NPM. Let me know if it works for you.

**Comment by visionpixel** [comment]
Nice, it works! Thanks.
